db/migrate/20170603022915_adds_issued_at_to_donation.rb

```diff
@@ -0,0 +1,14 @@
+class AddsIssuedAtToDonation < ActiveRecord::Migration[5.0]
+  # doin this old-school because we need to initialize it programmatically
+  def up
+  	add_column :donations, :issued_at, :datetime
+  	Donation.all.each do |d|
+  		d.issued_at = d.created_at
+  		d.save
+  	end
```

Comment:
Hm, good point. I've gotten stung by similar things before.

Seed migrations is interesting -- given the reliance on seed data maybe something like that would be useful?